packages/react/src/prefabs/PreJoin.tsx

```diff
@@ -312,7 +320,12 @@ export function PreJoin({
     }
 
     return () => {
-      videoTrack?.detach();
+      if (videoTrack) {
+        if (videoEl.current)
+          videoTrack.detach(videoEl.current);
+        videoTrack.stop();
```

Comment:
these changes are for `usePreviewDevice` not for `usePreviewTracks` - they seem unrelated to the issue?